config/sidekiq.yml

```diff
@@ -0,0 +1,5 @@
+---
+:concurrency: 5
+:queues:
+  - preservationIngestWF_default
+  - preservationIngestWF_low
```

Comment:
why is prescat the only set of robots with this?

Reply:
we used to configure similarly for common-accessioning, but from looking at that PR, seems like its `config/sidekiq.yml` did get moved to shared_configs?

if it doesn't change regularly or differ between environments, i think i'm personally indifferent?

Reply:
If it is the same across all workers for all environments, there is no need to put in shared configs. (Though one might make the case for putting it there anyway for consistency.)

Reply:
ah - yes, I like it here in the robot code if it's the same everywhere - but having it in both places is fine, too.